components/reducers/editAndDeleteReducer.js

```diff
@@ -0,0 +1,118 @@
+export const initialState = {
+  values: [],
+  attributes: [],
+  columns: [],
+  entity_type_name: null,
+  isLoading: false,
+  isRefresh: true,
+  isSaving: false,
+  isDeleting: false,
+  show: false,
+  entity_type_id: null,
+  entity_type_id_parent: null,
+  modalContent: null,
+};
+
+export const LOADING = 'LOADING';
```

Comment:
Kindly refactor this to a separate file. You can add it to reducers > actions.js